Remove acra-connector from timescale example

Lagovas opened the pull request:
* remove acra-connector usage
* update docker images to the newest versions
* configure timescale to use TLS certs
* update pgadmin configuration with pre-configured files

ZhmakaAS commented on timescaledb/scripts/entry.sh:
```diff
@@ -1,7 +1,11 @@
+export PGSSLCERT=/scripts/acra-client.crt
+export PGSSLKEY=/scripts/acra-client.key
+export PGSSLROOTCERT=/scripts/root.crt
+export PGSSLMODE=require
```
Why not `verify-full`?

Lagovas replied:
agree, will change

ZhmakaAS commented on _common/pgadmin/servers.json:
```diff
@@ -0,0 +1,16 @@
+    "1": {
+      "Name": "TimescaleDB",
+      "Host": "postgresql",
+      "Group": "Acra - TimescaleDB example",
```
Hm, is it only related to `TimescaleDB`, if I am not mistaken, pgAdmin image is using almost in all demos we have

Lagovas replied:
yeah, you are right. forgot about that. will change to some common `Acra - Postgresql example`.

ZhmakaAS commented on timescaledb/docker-compose.timescaledb.yml:
```diff
@@ -2,88 +2,60 @@ version: "3"
+            server-timescaledb:
+              # re-use ssl/pgsql/postgresql.crt with dns name 'postgresql'
+              aliases:
+                - "postgresql"
```
Cool, I didnt know about such approach:)

ZhmakaAS commented on timescaledb/docker-compose.timescaledb.yml:
```diff
@@ -93,89 +65,62 @@ services:
+            - grafana-server
+            - prometheus-server
+        ports:
+            - "9393:9393"
```
Do we need to open Acra ports to the host? I thought it is only for internal usage

Lagovas replied:
previously we exported acra-connector's ports to allow users connect via `psql` from host machine. there are same reason, to allow connect to acra not only by pgadmin, but by any other tool that can work with db: psql, DataGrid, etc

ZhmakaAS replied:
Got it, make sense

Lagovas commented on _common/pgadmin/pgadmin.dockerfile:
```diff
@@ -1,9 +0,0 @@
-FROM dpage/pgadmin4:4.29
```
yeah, you are right. I forgot about that and it has used in other examples... will check and update them too

Lagovas commented:
i'm bored with updating run.sh to run and check all other examples locally, so I refactored a bit run.sh to allow run it locally without downloading repo and without local patching run.sh file.
* now if you call `./run.sh django` from already cloned repository, then it just run locally this example
* if you run it with `curl <path> | bash -s -- django` then it will work too
* additionally I added avoiding failures of cloning related repos (acra repository for python example) if it is already exists (in case of local development when you didn't clean it before next run)
* updated alpine version in other dockerfiles and added required `py3-pip` package for new version
* removed `acra-engineering-demo` prefix for `COPY` commands to allow use it from local folder, not only cloned with `curl`, and changed context from `../../` to `../` to use relative from repo root path.
* also applied all suggestions:
* - change `require` to `verify-full` sslmode for timescale
* - renamed setting group for timescale to more common variant

so, @shadinua , @ZhmakaAS look one more time please due to much more changes

ZhmakaAS commented on django-transparent/django.dockerfile:
```diff
@@ -1,7 +1,4 @@
-FROM alpine:3.10
```
Hm, there was some explanation about newest alpine version and why it was not working with alpine:3.15.0. Have you checked if it is actually working demo with such base image version?

Lagovas replied:
it has worked, I checked. But anyway, updated hash commit for django project to version with newer psycopg2 version that fixes issues related to python3.8

ZhmakaAS reviewed (approved):
Lgtm, thanks
